components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -74,7 +74,7 @@ object LocaleManager {
      */
     fun getSystemDefault(): Locale {
         val config = Resources.getSystem().configuration
-        return ConfigurationCompat.getLocales(config).get(0)
+        return ConfigurationCompat.getLocales(config).get(0) ?: Locale.getDefault()
```

Comment:
Self-note: this is fine because `LocaleListCompatWrapper` has a range check and returns null otherwise instead of throwing an exception.